### ISSUE TITLE
did LaMA suite for mmwave radar sensor?

### ISSUE THREAD

**Issue by chopin1998** [iris-ua/iris_lama#21]

**Comment by eupedrosa** [comment]
TLDR: No.

A mmwave radar sensor is not yet suitable for SLAM. It may provide range data like a laser but the caracteristics of the data are very different. For example:
1. Short field-of-view
2. Low angular resolution + (range resolution vs maximum range)
3. Noisy + possible existence o multi-paths
4. Susceptible to interference by other mmwave radars.

Issue (1) can be handled by adding more mmwave sensors. I have a robot with 4, which provides a 360º fov. However, issue (4) is more visible. 
In issue (2), the radar trades maximum range by range resolution. More range, less resolution.
For (3), most of the noise can be filtered by looking at the intensity of each range detection, but it is not enough. For example, multi-paths created by metal can have high intensity.
In (4), the interference is higher when all radars have the same configurations (e.g. chirp slope). It is possible to address this problem by changing some configurations.

For these reasons, an mmwave radar does not work well with LaMa because on its reliance on scan matching.

**Comment by chopin1998** [comment]
thank thee very much!

what's central freq do your mmwave radar using? 66/ 77Ghz?

i have a ti 66Ghz radar, which can provide ~0.039m resolution, and ~ 10m+ range, i think it's ok for indoor application, right?

**Comment by eupedrosa** [comment]
>what's central freq do your mmwave radar using? 66/ 77Ghz?

77Ghz

> i have a ti 66Ghz radar, which can provide ~0.039m resolution, and ~ 10m+ range, i think it's ok for indoor application, right?

Yes.
